run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
What is this file? It feels like it's a local testing file mostly for your local debugging purposes, but maybe I'm wrong!

Reply:
It is. However it is generic enough that everybody would need to do these things one way or onother.

Reply:
I'm not sure I know what they do, so it's a bit difficult for me to understand how true that is. Please feel to treat like me an idiot and give me an explanation assuming I know nothing (I probably do know nothing in this case!) :)

Reply:
My reasoning is that nobody is going to build the repo inside of the cheri vm. Therefore it is useful to have a script that moves the executable and it's source code to the vm, and then runs it. It requires you to write multiple commands and it is easyer to put that in a shell script.

Reply:
I wonder if what we *really* want is a generic script (outside this repo) that says "run binary `X` in VM `Y`"?

Reply:
Removed the file will make a new repo for files like that.